src/main.rs

```diff
@@ -43,6 +43,11 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         .await
         .expect("Failed to create the permanent data directory");
 
+    log::info!(
```

Comment:
I'm not sure this is relevant because the connection to the Redis server is not made here, only the parsing of the input string.

Reply:
Having the config explicitly logged is a must imho, as it will show any configuration parsing or error almost instantly in the logs.
That being said, we might want to move it in the code, not sure where too, maybe right after the parsing?